### PR TITLE
Docs Update 2019-11

### PR DESCRIPTION
**Pull request by raduweiss**
## Reason for This PR

Update project root documentation files to reflect current state.

## Description of Changes

- Remove firecracker-devel@lists.infradead.org as a contact.
- Mention that Firecracker runs workloads at scale in AWS.
- Mention that AMD and Arm are in developer preview.
- Mention container integrations (Kata, Weave).
- Add the "KVM userspace" article to QEMU mentions.
- Mention rust-vmm as part of the crosvm story.
- Add OSv as a supported guest operating system.
- Remove OpenStack/CNCF FAQ question.
- Replace "security problem/vulnerability" with "security issue".
- Update maintainers.

## License Acceptance

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

## PR Checklist

`[Reviewer TODO: Verify that these criteria are met. Request changes if not]`

- [x] All commits in this PR are signed (`git commit -s`).
- [x] The reason for this PR is clearly provided.
- [x] The description of changes is clear and encompassing.
- [x] The required doc changes are included in this PR.
- [x] No code has been touched.
- [x] No API changes are included in this PR.
- [x] The changes in this PR have no usage impact.
- [x] No new `unsafe` code has been added.

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -34,43 +36,42 @@ to integrate with container ecosystems.
...
-keyboard controller used only to stop the microVM). This, along with a
-streamlined kernel loading process enables a < 125 ms startup time and a reduced
-memory footprint. The Firecracker process also provides a RESTful control API,
-handles resource rate limiting for microVMs, and provides a microVM metadata
-service to enable the sharing of configuration data between the host and guest.
+The Firecracker VMM is built to be processor agnostic. Intel processors are
+supported for production workloads. Support for AMD and Arm processors is in
+developer preview.
+
+### Can Firecracker be use dwithin the container ecosystem?
```
Comment (aghecenco) [inline FAQ.md]: Typo: used within

File: FAQ.md [hunk 2]
```diff
@@ -16,10 +15,13 @@ such as [AWS Lambda](https://aws.amazon.com/lambda/) and [AWS
...
 customer experience, while providing the security and isolation required of
 public cloud infrastructure. Firecracker started from Chromium OS's Virtual
-Machine Monitor (crosvm), an open source VMM written in Rust. Today, crosvm and
-Firecracker have diverged to serve very different customer needs. We plan to
-contribute bug fixes and tests for Rust crates that originated from crosvm, and
-any Firecracker functionality that's appealing for crosvm.
+Machine Monitor,
+[crosvm](https://chromium.googlesource.com/chromiumos/platform/crosvm/), an open
+source VMM written in Rust. Today, crosvm and Firecracker have diverged to
+serve very different customer needs. [Rust-vmm](https://github.com/rust-vmm) is
```
[the remark this answers is not in the document]
Comment (raduweiss) [reply FAQ.md]: We didn't actually end up contributing anything directly to crosvm, but instead created a new place (rust-vmm) to share code. I think it fits the narrative here.

File: README.md [hunk 3]
```diff
@@ -93,7 +93,8 @@ The **API endpoint** can be used to:
 - Configure the microvm by:
   - Setting the number of vCPUs (the default is 1).
   - Setting the memory size (the default is 128 MiB).
-  - Choosing a CPU template (currently, C3 and T2 are available).
+  - Choosing a CPU template (currently, C3 and T2 are available, the default is
```
[the remark this answers is not in the document]
Comment (raduweiss) [reply README.md]: I'll remove this then.

File: FAQ.md [hunk 4]
```diff
@@ -34,43 +36,42 @@ to integrate with container ecosystems.
...
+[Weave FireKube](https://www.weave.works/oss/firekube/) (via
+[Weave Ignite](https://github.com/weaveworks/ignite)), and containerd via
+[firecracker-containerd](https://github.com/firecracker-microvm/firecracker-containerd).
+We welcome contributions that enable Firecracker to integrate naturally with the
+container ecosystem and provide more choices in how container workloads are
+isolated.
+
+### What is the difference between Firecracker and QEMU?
+
+Firecracker is a cloud-native
```
[the remark this answers is not in the document]
Comment (raduweiss) [reply FAQ.md]: Ah indeed. I wasn't aware of that. I will update to "Firecracker is an alternative to QEMU that is purpose-built for running serverless functions and containers safely and efficiently, and nothing more."

File: FAQ.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (raduweiss) [reply FAQ.md]: Fixed.

**Pull request merged**